src/connector/src/common.rs

```diff
@@ -452,4 +490,17 @@ impl NatsCommon {
         let stream = jetstream.get_or_create_stream(config).await?;
         Ok(stream)
     }
+
+    pub(crate) fn create_credential(&self, seed: &str, jwt: &str) -> anyhow::Result<String> {
+        let creds = format!(
+            "-----BEGIN NATS USER JWT-----\n{}\n------END NATS USER JWT------\n\n\
+                         ************************* IMPORTANT *************************\n\
+                         NKEY Seed printed below can be used to sign and prove identity.\n\
+                         NKEYs are sensitive and should be treated as secrets.\n\n\
+                         -----BEGIN USER NKEY SEED-----\n{}\n------END USER NKEY SEED------\n\n\
+                         *************************************************************",
```

Comment:
incredible the func need the format

Reply:
Yes, this is what the nats documents said https://docs.rs/async-nats/latest/async_nats/struct.ConnectOptions.html#method.with_credentials